packages/contracts/evm-contracts/contracts/token/InverseAppProjectedNft.sol

```diff
@@ -75,7 +75,7 @@ contract InverseAppProjectedNft is IInverseAppProjectedNft, ERC721, Ownable {
     /// Increases the `totalSupply` and `currentTokenId`.
     /// Reverts if `_to` is a zero address or if it refers to smart contract but does not implement IERC721Receiver-onERC721Received.
     /// Emits the `Minted` event.
-    function mint(address _to, bytes calldata _verificationData) public virtual returns (uint256) {
+    function mint(address _to, bytes memory _verificationData) public virtual returns (uint256) {
```

Comment:
This no longer matches the interface

Reply:
Not true, quite the contrary, it **now** matches the interface